libkbfs/folder_branch_ops.go

```diff
@@ -569,6 +572,10 @@ func (fbo *folderBranchOps) setHeadLocked(ctx context.Context,
 		// Use uninitialized for the merged branch; the unmerged
 		// revision is enough to trigger conflict resolution.
 		fbo.cr.Resolve(md.Revision, MetadataRevisionUninitialized)
+	} else if md.MergedStatus() == Merged {
+		// if we are already merged through this write, the revision would be the
```

Comment:
Another super nit: comments that are complete sentences usually start in a capital letter and end in punctuation.